apps/lockfile-explorer-web/src/containers/LockfileViewer/index.tsx

```diff
@@ -155,11 +165,16 @@ export const LockfileViewer = (): JSX.Element | ReactNull => {
           }
         ]}
       />
-      <div className={`${appStyles.ContainerCard} ${styles.ViewWrapper}`}>
+      <div className={`${styles.ViewContents} ${appStyles.ContainerCard}`}>
         <div className={styles.LockfileFilterBar}>
-          {entryStack.length > 1 ? <button onClick={pop}>back</button> : null}
-          <h5>filter:</h5>
+          <h5>Filter:</h5>
           <input type="text" value={filter} onChange={updateFilter} />
+          <button disabled={entryStack.length > 1} onClick={pop}>
+            Back
+          </button>
+          <button disabled={entryForwardStack.length > 0} onClick={forward}>
+            Forward
+          </button>
         </div>
```

Comment:
![image](https://user-images.githubusercontent.com/4673363/202324923-fd5a05bb-e1ee-4f80-aedf-af14888adfe6.png)

Can we make it so that these checkboxes can be toggled by clicking the label ("Must have...")?  Currently it only works if you click inside the square glyph.

Reply:
Changed